build.gradle

```diff
@@ -326,6 +326,8 @@ tasks.register("integrationTest", Test) {
     useJUnitPlatform() {
         includeTags "integrationTest"
     }
+// Uncomment below to run unit tests in parallel
+//    maxParallelForks = Runtime.runtime.availableProcessors() * 2
```

Comment:
Parallel runs, when enabled do not seem to play well with in-jvm dtests. Since this was introduced in one of the prior commits in this PR, I will remove these commented lines for now. We can re-introduce it as an optimization.